web/src/features/charts/CarbonChart.tsx

```diff
@@ -50,6 +56,18 @@ function CarbonChart({ datetimes, timeAverage }: CarbonChartProps) {
         selectedTimeAggregate={timeAverage}
         tooltip={CarbonChartTooltip}
       />
+      <Divider />
+      <Accordion
+        title={t('data-sources.title')}
+        className="text-md"
+        isCollapsedAtom={dataSourcesCollapsedEmission}
+      >
+        <DataSources
+          title={t('data-sources.emission')}
+          icon={<IndustryIcon />}
+          sources={[...emissionSourceToProductionSource.keys()].sort()}
```

Comment:
I made this a Map instead of just a list so it would be easier to implement [AVO-84](https://linear.app/electricitymaps/issue/AVO-84/add-the-production-source-legend-under-data-sources-for-emission). When I do this I will also change the way emissiondata is found in the barbreakdownchart (barbreakdownchart line 96-105).